datahub-frontend/conf/routes

```diff
@@ -45,6 +45,8 @@ GET           /authenticate                                                   re
 POST          /logIn                                                          react.controllers.AuthenticationController.logIn()
 GET           /callback/oidc                                                  @org.pac4j.play.CallbackController.callback()
 POST          /callback/oidc                                                  @org.pac4j.play.CallbackController.callback()
+GET           /logOut                                                         @org.pac4j.play.LogoutController.logout()
```

Comment:
Do we think AuthenticationController should just extend LogOutController?

Reply:
Also, we'll need to call this endpoint when the "logout" button is clicked on the UI!